jax/_src/numpy/reductions.py

```diff
@@ -1224,10 +1224,83 @@ def nanprod(a: ArrayLike, axis: Axis = None, dtype: DTypeLike | None = None, out
                         axis=axis, dtype=dtype, out=out, keepdims=keepdims,
                         initial=initial, where=where)
 
-@implements(np.nanmean, skip_params=['out'])
+
 @partial(api.jit, static_argnames=('axis', 'dtype', 'keepdims'))
 def nanmean(a: ArrayLike, axis: Axis = None, dtype: DTypeLike | None = None, out: None = None,
             keepdims: bool = False, where: ArrayLike | None = None) -> Array:
+  r"""Return the mean of the array elements along a given axis, ignoring NaNs.
+
+  JAX implementation of :func:`numpy.nanmean`.
+
+  Args:
+    a: Input array.
+    axis: int or sequence of ints, default=None. Axis along which the mean is
+      computed. If None, the mean is computed along the flattened array.
+    dtype: The type of the output array. Default=None.
+    keepdims: bool, default=False. If True, reduced axes are left in the result
+      with size 1.
+    initial: int or array, default=None. Initial value for the mean.
```

Comment:
There is no `initial` parameter for `nanmean`

Reply:
Sorry, it was included by mistake. Removed.